spec/acceptance/clock_spec.rb

```diff
@@ -75,5 +75,13 @@
 
       it { should_not have_run("Run at certain time") }
     end
+
+    context 'when skip_first_run is present' do
```

Comment:
Minor quibble, but the rest of the codebase has used double-quotes for all strings, and I appreciate consistency when we can have it :-).